mujoco-sys/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "mujoco-rs-sys"
 version = "0.0.2"
-authors = ["Ryan Butler <thebutlah@gmail.com>", "Aaron Law <aaronlaw1215@gmail.com>"]
+authors = [
+    "Ryan Butler <thebutlah@gmail.com>",
+    "Aaron Law <aaronlaw1215@gmail.com>",
```

Comment:
You should add yourself here actually.  You've done a lot for this effort, you should get credit.